int-tests/src/test-integration/java/io/neow3j/compiler/HashIntegrationTest.java

```diff
@@ -257,6 +271,14 @@ public static Hash256 hash256FromStringLiteral() {
             return new Hash256(
                     StringLiteralHelper.hexToBytes("03b4af8d061b6b320cce6c63bc4ec7894dce107b000000000000000000000000"));
         }
+
+        public static io.neow3j.devpack.Hash160 hash160FromString() {
+            return new io.neow3j.devpack.Hash160("0xcc5e4edd9f5f8dba8bb65734541df7a1c081c67b");
```

Comment:
Do we also support strings without the "0x" prefix?

Reply:
Yes, a `types.Hash160` is initialised when compiling that string. So, the prefix does not matter. See here: https://github.com/neow3j/neow3j/blob/f45ce15d23cdfc6820a95c97b54d601147aa06d7/compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java#L334-L339